src/test/java/it/pagopa/pn/deliverypush/service/impl/PaperChannelServiceImplTest.java

```diff
@@ -457,7 +457,7 @@ void sendNotificationAttachments_CfgNone() {
         paperChannelService.sendSimpleRegisteredLetter(notificationInt, 0, "req123", physicalAddressInt, "NR_SR", Collections.emptyList());
 
         // THEN
-        Mockito.verify(attachmentUtils, Mockito.never()).getNotificationAttachments(any());
+        Mockito.verify(attachmentUtils, Mockito.never()).getNotificationAttachments(any(), any());
         Mockito.verify(attachmentUtils, Mockito.never()).getNotificationAttachmentsAndPayments(any(), any(), anyInt(), anyBoolean(), any());
```

Comment:
Qui aggiungere il verify del popolamento della data di sentAt.
dentro a newNotification (riga 444)  quando costruisce l'oggetto impostagli pure una data (no instant now, mettici na data fissa in modo che poi qui sotto riesci a testarla)